### PR TITLE
Task/prettier for markdown

### PR DESCRIPTION
**Pull request by nickytonline**
## What type of PR is this? (check all applicable)
- [x] Refactor
- [ ] Feature
- [ ] Bug Fix

## Description
Implements #334.

Since we already run prettier on other files, we might as well run it on markdown files as well.

## Added to documentation?
  - [x] docs.dev.to
  - [x] readme
  - [ ] no documentation needed

## [optional] What gif best describes this PR or how it makes you feel?

f2046b27-7982-4503-90ee-c50173a723d8

### CODE REVIEW

File: docs/additional-postgres-setup.md [hunk 1]
```diff
@@ -1,16 +1,18 @@
 # Setup your application with PostgreSQL
+
 Follow these links for detailed the installation guide.
```
Comment (ocnly) [inline docs/additional-postgres-setup.md]: This line doesn't sound right.

Comment (nickytonline) [reply docs/additional-postgres-setup.md]: You're right it doesn't, although I wasn't looking at what was written, just running prettier. I can update that line in the text though.

File: docs/additional-postgres-setup.md [hunk 2]
```diff
@@ -1,16 +1,18 @@
 # Setup your application with PostgreSQL
-Follow these links for detailed the installation guide.
 
-1. [Mac OS](https://postgresapp.com/)
-2. Linux / Ubuntu
-   - [Ubuntu `14.04`](https://www.digitalocean.com/community/tutorials/how-to-install-and-use-postgresql-on-ubuntu-14-04)
-   - [Ubuntu `16.04 and higher`](https://www.digitalocean.com/community/tutorials/how-to-install-and-use-postgresql-on-ubuntu-16-04)
-3. [Windows](https://www.postgresql.org/download/windows/)
+Follow the instructions in the the installation guide below that corresponds to your operating system.
```
Comment (nickytonline) [inline docs/additional-postgres-setup.md]: @ocnly, I've updated the wording.

File: docs/additional-postgres-setup.md (hunk 1 above)
Comment (ocnly) [reply docs/additional-postgres-setup.md]: Oh I was oblivious to the fact that this was not one of your changes, my bad :(

File: docs/additional-postgres-setup.md (hunk 2 above)
Comment (ocnly) [reply docs/additional-postgres-setup.md]: That's good, but a duplicate definite article managed to slip in though. Other than that, this sounds much better. Thanks.

File: docs/additional-postgres-setup.md (hunk 1 above)
Comment (nickytonline) [reply docs/additional-postgres-setup.md]: No worries ;)

File: docs/additional-postgres-setup.md (hunk 2 above)
Comment (nickytonline) [reply docs/additional-postgres-setup.md]: Done. Got rid of my double-vision. 👓

**Pull request merged**